easybuild/easyconfigs/s/SWIG/SWIG-4.0.2-GCCcore-10.2.0.eb

```diff
@@ -1,3 +1,7 @@
+# Contribution from the NIHR Biomedical Research Centre
```

Comment:
Please remove the addition of authorship over this - since `SWIG-4.0.2-GCCcore-10.2.0.eb` is already in EasyBuild.

Reply:
I have removed the authorship for both files as HTSeq is also already in EasyBuild.